cylc/flow/scheduler.py

```diff
@@ -616,7 +614,6 @@ async def run(self):
 
         """
         try:
-            await self.install()
```

Comment:
`Scheduler.install()` has already just been run in `scheduler_cli._setup()` https://github.com/cylc/cylc-flow/blob/dd8e26ac9cbba3c4b4683380456e59c52e000d8d/cylc/flow/scheduler_cli.py#L293-L310

https://github.com/cylc/cylc-flow/blob/dd8e26ac9cbba3c4b4683380456e59c52e000d8d/cylc/flow/scheduler_cli.py#L334-L337

https://github.com/cylc/cylc-flow/blob/dd8e26ac9cbba3c4b4683380456e59c52e000d8d/cylc/flow/scheduler_cli.py#L342-L347

Reply:
Hmmm, why the heck was it like that.

Would it work to keep the install in the Scheduler and remove it from `scheduler_cli`?

Reply:
I tried removing `scheduler_cli._setup()` first, but the
```python
 # setup loggers 
 _open_logs(reg, options.no_detach) 
```
bit fails without the workflow run dir tree set up

Reply:
Dammit, nvm.